lab_1/main.py

```diff
@@ -20,6 +20,16 @@ def tokenize(text: str) -> list:
 
 
 def remove_stop_words(tokens: list, stop_words: list) -> list:
+    def remove_stop_words(tokens: list, stop_words: list) -> list:
```

Comment:
why def with \tab?
why function description without \tab?

put this description in EVERY function